onnxruntime/core/providers/ngraph/ngraph_execution_provider.cc

```diff
@@ -274,7 +320,7 @@ static std::map<std::string, std::set<std::string>> GetNgSupportedOps(const int
   std::map<std::string, std::set<std::string>> ng_supported_ops;
   ng_supported_ops.emplace(kOnnxDomain, ngraph::onnx_import::get_supported_operators(onnx_opset, kOnnxDomain));
 
-  const std::set<std::string> ng_disabled_ops = {};  //Place-holder for ops not supported.
+  const std::set<std::string> ng_disabled_ops = {"LSTM", "Gather"};  //Place-holder for ops not supported.
```

Comment:
Just wanted to confirm. We saw poor performance for LSTM in nGraph 0.18, is it still the case in 0.22?

Re: Gather. I think the issue is actually that Gather is supported by nGraph but not for int8 inputs. So instead of disabling Gather completely, we may want to only disable for int8 type.

Reply:
About Gather - I've tried disabling int8 inputs, then anything else than float but it was still not performing well. This is already investigated by the cpu-backend team and I'm afraid we'll have to disable it entirely for 0.22.1 version of nGraph.